extensions/podman/src/extension.ts

```diff
@@ -578,7 +578,7 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   // allows to create machines
   if (isMac() || isWindows()) {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const createFunction = async (params: { [key: string]: any }, logger: extensionApi.Logger): Promise<void> => {
+    const createFunction = async (params: { [key: string]: any }, logger: extensionApi.Logger): Promise<number> => {
```

Comment:
I guess that if you need something, you should return a complex object with like `pid` field in the returned object
Because I suppose at some point someone else will want to return an additional information.

Reply:
you mean the childProcess itself?